projectTemplates/js-server/.eslintrc.json

```diff
@@ -13,7 +15,7 @@
     "space-before-function-paren": "off",
     "indent": [
       "error",
-      2
+      4
```

Comment:
I think 2 is better lets set a vote, or we can set it to 2 but the people in your team can change it later to 4 or the vice versa, I don't really care.

Reply:
changed